common/src/main/scala/io/phdata/services/AccountServiceImpl.scala

```diff
@@ -52,7 +52,7 @@ class AccountServiceImpl[F[_]: Sync: Timer](
 
   override def ldapAuth(username: String, password: Password): OptionT[F, Token] =
     for {
-      user <- context.lookupLDAPClient.validateUser(username, password)
+      user <- context.provisioningLDAPClient.validateUser(username, password)
```

Comment:
Why did this change?

Reply:
because of above change, method `validateLDAPClient` is now only member of `ProvisioningLDAPClient` so it cannot be called on `LookupLDAPClient`

Reply:
We need to keep the old behavior, since we know it works you can move  `validateUser`  to the lookup client.